### PR TITLE
[test/features] Add test for show features command

### PR DESCRIPTION
**Pull request by pra-moh**
Description of PR
Verify show features command output against CONFIG_DB
image

CONFIG_DB output:
image

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ *] Test case(new/improvement)

#### How did you verify/test it?
Using sonic virtual test bed
image

#### Any platform specific information?
Applicable for 201911 image

### CODE REVIEW

File: tests/test_features.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import pytest
+import logging
+
+logger = logging.getLogger(__name__)
```
Comment (jleveque) [inline tests/test_features.py:4]: logger is unused, unless you want to keep it around for future enhancements

File: tests/test_features.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    """Verify show features command output against CONFIG_DB
+    """
+    features_stdout = duthost.shell('show features', module_ignore_errors=True)['stdout_lines']
+    features_dict = get_dict_stdout(features_stdout)
+    for k,v in features_dict.items():
+        feature = str(k)
+        status_out = duthost.shell('/usr/bin/redis-cli -n 4 hgetall "FEATURE|{}"'.format(feature), module_ignore_errors=False)['stdout_lines']
+        redis_value = get_status_redisout(status_out)
+        if str(redis_value) == str(v):
+            assert True, "{} is {} which matches with config_db".format(k,v)
```
Comment (jleveque) [inline tests/test_features.py:37]: Suggest to add newline at EOF

File: tests/test_features.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    status_list = status_out[1:]
+    status = ""
+    for s in status_list:
+        status = s.encode('UTF-8')
+        return status
+
+def test_show_features(duthost):
+    """Verify show features command output against CONFIG_DB
+    """
+    features_stdout = duthost.shell('show features', module_ignore_errors=True)['stdout_lines']
```
Comment (lguohan) [inline tests/test_features.py:30]: module_ignore_errors=False

File: tests/test_features.py (hunk 1 above)
Comment (pra-moh) [reply tests/test_features.py]: removed

File: tests/test_features.py (hunk 3 above)
Comment (pra-moh) [reply tests/test_features.py]: changed to False

File: tests/test_features.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import pytest
+import logging
```
Comment (jleveque) [inline tests/test_features.py:2]: These imports are unused. Remove?

File: tests/test_features.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    """Verify show features command output against CONFIG_DB
+    """
+    features_stdout = duthost.shell('show features', module_ignore_errors=False)['stdout_lines']
+    features_dict = get_dict_stdout(features_stdout)
+    for k,v in features_dict.items():
+        feature = str(k)
+        status_out = duthost.shell('/usr/bin/redis-cli -n 4 hgetall "FEATURE|{}"'.format(feature), module_ignore_errors=False)['stdout_lines']
+        redis_value = get_status_redisout(status_out)
+        if str(redis_value) == str(v):
+            assert True, "{} is {} which matches with config_db".format(k,v)
```
Comment (hui-ma) [inline tests/test_features.py:34]: question: will the assert line shown in the test output both for good case or bad case? or only shown in the failure case?

Comment (jleveque) [reply tests/test_features.py:34]: The message is the assert line will only be printed if the condition fails. Thus, this message is incorrect, it should report the failure, not the success.

File: tests/test_features.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+def get_status_redisout(status_out):
+    """Extract status value for feature in redis
+    """
+    status_list = status_out[1:]
+    status = ""
+    for s in status_list:
+        status = s.encode('UTF-8')
+        return status
+
+#Test Functions
```
Comment (jleveque) [inline tests/test_features.py:23]: Add space after `#`

File: tests/test_features.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+def test_show_features(duthost):
+    """Verify show features command output against CONFIG_DB
+    """
+    features_stdout = duthost.shell('show features', module_ignore_errors=False)['stdout_lines']
+    features_dict = get_dict_stdout(features_stdout)
+    for cmd_key,cmd_value in features_dict.items():
```
Comment (jleveque) [inline tests/test_features.py:29]: Add space after `,`

File: tests/test_features.py (hunk 5 above)
Comment (pra-moh) [reply tests/test_features.py]: updated

File: tests/test_features.py (hunk 6 above)
Comment (pra-moh) [reply tests/test_features.py]: added

File: tests/test_features.py (hunk 7 above)
Comment (pra-moh) [reply tests/test_features.py]: added

**Pull request merged**